src/schema/apple/episodeDetails.json

```diff
@@ -38,17 +38,7 @@
           "type": "number",
           "default": 0
         }
-      },
```

Comment:
I think we can add this again, if we convince ajv to not only validate the schema but also edit the data:
see https://ajv.js.org/options.html#usedefaults

maybe we can add this in `src/schema/SchemaValidator.ts`

Reply:
true; added the option and reverted the schema changes. Tested and works